sleepecg/config.py

```diff
@@ -22,15 +22,16 @@ def _read_yaml(path: Path) -> Dict[str, Any]:
     try:
         with open(path) as file:
             cfg = yaml.safe_load(file)
-            # empty .yml-files are loaded as `None`
-            if cfg is None:
-                return {}
-            if not isinstance(cfg, dict):
-                raise ValueError(f'Invalid YAML config file at {_USER_CONFIG_PATH}')
-            return cfg
     except FileNotFoundError:
         return {}
 
+    # empty .yml-files are loaded as `None`
+    if cfg is None:
+        return {}
+    if not isinstance(cfg, dict):
```

Comment:
What on earth gets returned from an invalide YAML file?

Reply:
If it's actually syntactically invalid, PyYAML will raise a `ParserError`... but a syntactically valid YAML file might contain a list/str/int/float instead of a mapping, which wouldn't make too much sense for our usecase^^

Reply:
Right, I guess that's OK then.